source/base_gui.d

```diff
@@ -186,6 +176,7 @@ class BaseGui
                 gp._vbo.setData(vp.vertices());
 
             gp._indices = iota(0, vp.vertices.length).map!"cast(uint)a".array;
+            gp._ibo.setData(gp._indices);
```

Comment:
No, but in one way or another it needs a refactoring : no encapsulating at all. Will be removed in next PR.